superset-frontend/src/SqlLab/components/TableElement/index.tsx

```diff
@@ -273,77 +242,90 @@ const TableElement = ({ table, ...props }: TableElementProps) => {
       );
     }
     return (
-      <ButtonGroup
-        css={css`
-          column-gap: ${theme.sizeUnit * 1.5}px;
-          margin-right: ${theme.sizeUnit}px;
-          & span {
-            display: flex;
-            justify-content: center;
-            width: ${theme.sizeUnit * 4}px;
-          }
-        `}
-      >
-        <IconTooltip
-          className="pull-left m-l-2 pointer"
-          onClick={refreshTableMetadata}
-          tooltip={t('Refresh table schema')}
-        >
-          <Icons.SyncOutlined
-            iconSize="m"
-            iconColor={theme.colors.primary.dark2}
-          />
-        </IconTooltip>
-        {keyLink}
-        <IconTooltip
-          className={
-            `fa fa-sort-${sortColumns ? 'numeric' : 'alpha'}-asc ` +
-            'pull-left sort-cols m-l-2 pointer'
-          }
-          onClick={toggleSortColumns}
-          tooltip={
-            sortColumns
-              ? t('Original table column order')
-              : t('Sort columns alphabetically')
-          }
-        />
-        {tableData.selectStar && (
-          <CopyToClipboard
-            copyNode={
+      <StyledControls>
+        {isMetadataFetching || isExtraMetadataLoading ? (
+          <Loading position="inline" />
+        ) : (
+          <Fade
+            data-test="fade"
+            hovered={hovered}
+            onClick={e => e.stopPropagation()}
+          >
+            <ButtonGroup
+              css={css`
+                column-gap: ${theme.sizeUnit * 1.5}px;
+                margin-right: ${theme.sizeUnit}px;
+
+                & span {
+                  display: flex;
+                  justify-content: center;
+                  width: ${theme.sizeUnit * 4}px;
+                }
+              `}
+            >
+              <IconTooltip
+                className="pull-left m-l-2 pointer"
+                onClick={refreshTableMetadata}
+                tooltip={t('Refresh table schema')}
+              >
+                <Icons.SyncOutlined
+                  iconSize="m"
+                  iconColor={theme.colors.grayscale.dark5}
+                />
+              </IconTooltip>
+              {keyLink}
               <IconTooltip
-                aria-label="Copy"
-                tooltip={t('Copy SELECT statement to the clipboard')}
+                className={
+                  `fa fa-sort-${sortColumns ? 'numeric' : 'alpha'}-asc ` +
```

Comment:
We are removing usages of fa icons from the codebase. Can you use the Icons component from `src/components/Icons `instead?